Cargo.toml

```diff
@@ -22,7 +22,7 @@ rand="0.7"
 byteorder="1.2"
 lru-cache = "0.1.1"
 bitcoin_hashes = "0.7"
-bitcoin = { version= "0.21", features=["serde"], optional = true }
+bitcoin = { git = "https://github.com/rust-bitcoin/rust-bitcoin", rev="e8bcde4", features=["use-serde"], optional = true }
```

Comment:
Why is this using a git branch instead of a release?

Reply:
I think the reason is that at the time this PR was create there wasn't a recent tag available, but I see `0.24.0` was just tagged a couple weeks ago, so probably should be using that one now.